ui/i18n.js

```diff
@@ -0,0 +1,31 @@
+
+// cross-browser connection to extension i18n API
+
+const chrome = chrome || null
+const browser = browser || null
+let getMessage = null
+
+if ((chrome && chrome.i18n && chrome.i18n.getMessage) ||
+    (browser && browser.i18n && browser.i18n.getMessage)) {
+  getMessage = (chrome || browser).i18n.getMessage
+} else {
+  // fallback function
+  console.warn('browser.i18n API not available?')
+  const msg = require('../app/_locales/en/messages.json')
```

Comment:
This looks hard-coded to English. I guess we would need to manually change this and rebuild per language?

Reply:
Oh this is just the fallback function. I'm still looking for where the conditional swap is.